pages/api/v1/sessions/index.public.js

```diff
@@ -43,43 +43,63 @@ function postValidationHandler(request, response, next) {
 }
 
 async function postHandler(request, response) {
+  const startTime = process.hrtime.bigint();
+  const targetTime = Math.random() * 1000 + 2500;
```

Comment:
Como existe a parte randômica, não se justifica a precisão do `hrtime`:

```suggestion
  const startMs = Date.now();
```

A outra variável só é necessária se for precisar aguardar.